crates/rolldown_plugin_wasm/src/lib.rs

```diff
@@ -20,7 +19,7 @@ impl Plugin for WasmPlugin {
   async fn resolve_id(
     &self,
     _ctx: &SharedPluginContext,
-    args: &HookResolveIdArgs,
+    args: &HookResolveIdArgs<'_>,
   ) -> HookResolveIdReturn {
```

Comment:
This is wonderful. Though we define the `Plugin` trait in desurgared form, which is without `async` keyword. But rustc is ok to write `async` keyword in the `impl XXX` position, which save us from adding meaningless `-> impl Future<Output=HookResolveIdReturn> + Send`.

Reply:
And this is an expected behavior: https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/Auto.20infered.20.60Send.60.20in.20.60impl.20DesugaredAsync.60.2E.20Expected.3F